### DISCUSSION TITLE
Multiple mobile devices using same Immich account possible?

### DISCUSSION THREAD

**Discussion by OlegLoewen** [immich-app/immich#4951] [Q&A]
Hello, as a consolidated view of partner photos and mine is currently not available (https://github.com/immich-app/immich/discussions/2846), I was wondering if it would work, if my wife would use my Immich account on her phone as well. 
So I tried it out and here is my observation.
* both devices are uploading photos to the server
* on web and on my mobile phone I can see the photos, that have been uploaded from device of my wife
* **BUT** on my wife's phone only local photos are shown !? Is this a bug, or expected behavior?

**Comment by alextran1502** [comment]
Consolidating view is being worked on at the moment and should be available in the next release 

https://github.com/immich-app/immich/pull/4933

**Reply by AndyPro720** [reply]
@alextran1502 quick question, do these assets show up in yearly recall? If I recall correctly, they don't show up in faces and maps(as of now), right?

Also, thanks a ton for this implementation! :^)
